### DISCUSSION TITLE
Problem with key-prop

### DISCUSSION THREAD

**Discussion by gulfemakkaya** [neuefische/questions#487] [Web] [answered]
Hi everyone! 

I have an array of objects which all have an id . Even though I´m calling it in my Index.js it's not getting accepted and still gives me an error.

`<ListItem key={plant.id}>`

Warning: Each child in a list should have a unique "key" prop.

Check the render method of `PlantCard`.

Thanks ahead for your help!

**Comment by niklasrammelkamp** [comment]
Hi Gülfem, can you post the whole map function and your plant objects? 
Seems like your objects have no ids or some of them have same.

**Reply by doemser** [reply]
Can you also remove duplicate Discussion? This does not seem intentional.

EDIT: Found out that I can delete discussions, so I used my power directly. :P

**Reply by gulfemakkaya** [reply]
I had a problem with my internet connection which lead to these duplicates 😬
Thank you for using your powers :D

**Comment by gulfemakkaya** [comment, marked as answer]
BEFORE: 
```js
return (
            <>
              <ListItem key={plant.id}>
                <h2>{plant.name}</h2>

                {
                  <Image
                    src={plant.image}
                    width={100}
                    height={100}
                    alt={plant.name}
                  />
                }
              </ListItem>
            </>
          );
        })}
```

AFTER: 
  ```js
return (
            
              <ListItem key={plant.id}>
                <h2>{plant.name}</h2>

                {
                  <Image
                    src={plant.image}
                    width={100}
                    height={100}
                    alt={plant.name}
                  />
                }
              </ListItem>
            
          );
        })}
```

**Reply by doemser** [reply]
Oh okay, so you removed the Fragments and now it works?

**Reply by gulfemakkaya** [reply]
Jipp, the Fragments were the problem!
